tools/probers/bazelrbe/bazelrbe.go

```diff
@@ -63,6 +64,14 @@ func createWorkspace(dir string, numTargets, numInputsPerTarget, inputSizeBytes
 		return err
 	}
 
+	if *proberName != "" {
+		dir = dir + "/" + *proberName
+		err := os.Mkdir(dir, 0777)
```

Comment:
doesn't matter much here but 0755 is typically a better default for directories

Reply:
Done